fileattrs/os2.attr

```diff
@@ -0,0 +1,3 @@
+%__os2_provides		%{_rpmconfigdir}/os2deps.sh --provides
+%__os2_requires		%{_rpmconfigdir}/os2deps.sh --requires
+%__os2_magic		^(32|64)-bit.*$
```

Comment:
This magic is too broad. Is there a may to make this a more restrictive match so that it doesn't just get everything?

Reply:
The full output is 

     32-bit OS/2 executable [WINDOWCOMPAT]

so maybe I can add 'OS/2'

Reply:
mmhh is it ran across which file types? e.g. DLL  magic is

    32-bit DLL [WINDOWCOMPAT]

Reply:
Would this work? `^(32|64)-bit (OS\/2 executable|DLL).*$`

Reply:
that would be fine, sure.

Reply:
I haven't tested it, so you should probably test. :)